packages/hash/frontend/src/shared/layout/layout-with-header/actions-dropdown.tsx

```diff
@@ -114,16 +114,24 @@ export const ActionsDropdownInner: FunctionComponent<{
         >
           <ListItemText primary="Create Page" />
         </MenuItem>
-        {activeWorkspace ? (
-          <>
-            <MenuItem href="/new/entity" onClick={popupState.close}>
-              <ListItemText primary="Create Entity" />
-            </MenuItem>
-            <MenuItem href="/new/types/entity-type" onClick={popupState.close}>
-              <ListItemText primary="Create Entity Type" />
-            </MenuItem>
-          </>
-        ) : null}
+        {activeWorkspace
+          ? [
```

Comment:
Fixes this console warning (which fail Playwright tests because of #436)

```
[error] MUI: The Menu component doesn't accept a Fragment as a child. Consider providing an array instead.
```